mitodl/mitxonline
# Use poetry for python dependency management

# What are the relevant tickets?
Closes #1753 

# Description (What does it do?)
Removes requirements files, uses poetry to specify & install requirements

# How can this be tested?
Run `docker-compose build --no-cache web celery`
CI tests should pass, everything should still work.  
Try registering a new account, enrolling in a course, etc
Also try running docker commands like `docker-compose run --rm web pytest authentication/api_test.py`
